testing/cmake/write_git_revision-custom-prefix/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 #=============================================================================
 include(${rapids-cmake-dir}/cmake/write_git_revision_file.cmake)
 
-cmake_minimum_required(VERSION 3.20)
+cmake_minimum_required(VERSION 3.23)
```

Comment:
Is there a reason that tests are not consistent with the source in requiring the patch version?

Reply:
The regex I use searched for `.20` and therefore I missed bumping the tests correctly.
I have updated all the tests to 3.23.1